content/applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst

```diff
@@ -1,197 +1,158 @@
-serial number configuration to a specific website in the :guilabel:`Website` field (if working in a
-multi-website environment).
+Assigning new or existing lot numbers to incoming goods can be done directly on receipts.
 
-A detailed description of this specific lot or serial number can also be added in the
-:guilabel:`Description` tab below.
+To begin, go to the :menuselection:`Purchase` app to create and confirm a |PO| for products tracked
+by lot numbers. Then, select the :guilabel:`Receipt` smart button that appears at the top of the
+page to navigate to the warehouse receipt form.
 
-When all desired configurations are complete, click the :guilabel:`Save` button to save all changes.
-
-.. image:: lots/lots-new-lot-number.png
-   :align: center
-   :alt: New lot number creation form with assigned product.
-
-After a new lot number has been created, saved, and assigned to the desired product, navigate back
-to the product form in the :menuselection:`Inventory` app, by going to :menuselection:`Products -->
-Products`, and selecting the product to which this newly-created lot number was just assigned.
+.. note::
+   Alternatively, navigate to an existing receipt by going to the :menuselection:`Inventory` app,
+   selecting the :guilabel:`Receipts` card, and choosing the desired receipt.
 
-On that product's detail form, click the :guilabel:`Lot/Serial Numbers` smart button to view the new
-lot number. When additional quantity of this product is received or manufactured, this new lot
-number can be selected and assigned to it.
+.. important::
+   Clicking :guilabel:`Validate` before assigning a lot number triggers a :guilabel:`User Error`,
+   indicating that a lot number **must** be assigned before validating the receipt.
 
-Manage lots for shipping and receiving
-======================================
+   .. image:: lots/user-error.png
+      :align: center
+      :alt: Add lot/serial number user error popup.
 
-Lot numbers can be assigned for both **incoming** and **outgoing** goods. For incoming goods, lot
-numbers are assigned directly on the purchase order form. For outgoing goods, lot numbers are
-assigned directly on the sales order form.
+On the receipt form, select the :guilabel:`⦙≣ (bulleted list)` icon to the right of the product that
+is tracked by lot numbers.
 
-Manage lots on receipts
------------------------
+.. figure:: lots/bulleted-list-icon.png
+   :align: center
+   :alt: Show the bulleted list icon on the product line.
 
-Assigning lot numbers to **incoming** goods can be done directly from the purchase order (PO).
+   Show the bulleted list icon on the product line.
 
-To create a :abbr:`PO (purchase order)`, go to :menuselection:`Purchase app --> Create`. Doing so
-reveals a new, blank request for quotation (RFQ) form.
+Doing so opens the :guilabel:`Detailed Operations` pop-up window, where the :guilabel:`Lot/Serial
+Number` and quantity are assigned.
```

Comment:
Should this be "guilabel:`Done` quantity" instead of "quantity"? (Leaving this up to you because I honestly don't know for sure one way or the other) :)